co-log/example/Main.hs

```diff
@@ -1,26 +1,30 @@
-{-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE FlexibleContexts  #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeApplications  #-}
 
 module Main where
 
-import Colog (WithLog, cmap, logMsg, logStringStderr, logStringStdout, usingLoggerT, withLog,
-              withLogStringFile)
+import Colog (LogMessage (..), WithLog, cmap, fmtLogMessage, logDebug, logInfo, logMsg,
+              logTextStderr, logTextStdout, logWarning, usingLoggerT, withLog, withLogTextFile)
 
-example :: WithLog env String m => m ()
+example :: WithLog env LogMessage m => m ()
 example = do
-    logMsg "First message..."
-    logMsg "Second message..."
+    logDebug "First message..."
+    logInfo "Second message..."
```

Comment:
I would like to ask you what style of logging do you prefer?

1. `logDebug "Message"`
2. `log Debug "Message"`.

The first style is one character shorter. But you need to import a lot of different functions if you're using logging severity of different levels: 
```haskell
import Colog (logDebug, logWarn, logInfo, logError, logNotice)
```
The second style is one character longer, but imports are shorter:
```haskell
import Colog (Severity (..), log)
```

Maybe there're other cons and pros of these styles? 🤔 But even if we stick to single style, it might be a good idea to provide to users functions of different types.

Reply:
Yes, I'm not sure which I like more, I just get used to the style above, but I agree that we should give users the opportunity to choose their favourite way, it won't harm.